server/models/handlers/signup.js

```diff
@@ -0,0 +1,121 @@
+import java.util.ArrayList;
+ import java.util.Scanner;
+
+
+ public class signup {
+
+        public static void  main(String[] args) {
+           
+           //Create arrays to store the credentials provided by the user 
+        //Array for users who are students
+        ArrayList<userType> users = new ArrayList<userType>(); 
+        //Array for users who are serviceStaff
+        ArrayList<userType> serviceStaff = new ArrayList<userType>(); 
+            
+            //Create a scanner to injest the credentials which the user inputs
+
+            Scanner input = new Scanner(System.in); 
+
+            String email, accessToken
+            , schoolName;
+
+            //The main account creation page
+
+            System.out.println("<<<WELCOME TO THE ACCOUNT CREATION PAGE>>>");
+            
+            /*There are different sign up experiences for students and service staff. 
+            The following asks the user to specify if they are a student or a service staff*/
+
+            System.out.println("Are you a student (type '1') or a service staff (type '2')?:  ");
+            int userTypeSelection = input.nextInt();
+
+/* Based on the above input, the system executes either case 1 for a student user 
+or case 2 for a service staff user*/
+
+
+            /* In case 1, The system asks the student users to input their credentials 
+            which are their 
+            firstName
+            lastName
+            email
+            accessToken
+            
+            phone
+            schoolID.*/
+
+switch (userTypeSelection) {
+
+    case 1:
+    System.out.println("<<Student Sign Up>>");
+
+    System.out.print("firstName : "); 
+    email = input.next();
+
+    System.out.print("lastName : "); 
+    email = input.next();
+
+    System.out.print("email : "); 
+    email = input.next();
+
+    System.out.print("accessToken
```

Comment:
The user would not know their access token and this value would need to be generated randomly using an npm library such as randomstring (https://www.npmjs.com/package/randomstring)

Reply:
I see, I'll change this so an authentication token is generated as an output of this logic.

Reply:
Would the user need to select a password and input that through the sign up form?